ethereum_history_api/oracles/src/script/noir_fixtures/boundedVec.ts

```diff
@@ -14,10 +14,10 @@ export function createFocusedBoundedVecFixture(value: Hex): string {
 }`;
 }
 
-export function createVerticalBoundedVecFixture(values: string[]): string {
+export function createVerticalBoundedVecFixture(values: string[], len: number = values.length): string {
```

Comment:
Agree this can be more consistent. I propose to to it in a separate PR. Created task for simplifying this API https://www.notion.so/vlayer/Make-create-BoundedVecFixture-functions-interface-consistent-509ae23e52e94d8ab845be01da0563e0?pvs=4